### PR TITLE
Set monitor default geometry on Linux

### PR DESCRIPTION
**Pull request by jwerle**
https://github.com/socketsupply/socket/issues/363

### CODE REVIEW

File: src/window/linux.cc [hunk 1]
```diff
@@ -633,9 +641,25 @@ namespace SSC {
...
 
-      gdk_monitor_get_workarea(monitor, &geometry);
-      width = (int) geometry.width;
-      height = (int) geometry.height;
+      if (monitor) {
+        gdk_monitor_get_workarea(monitor, &geometry);
+      }
+
+      if (geometry.width > 0) {
+        width = (int) geometry.width;
```
Comment (chicoxyzzy) [inline src/window/linux.cc:649]: it is `int` already https://docs.gtk.org/gdk3/struct.Rectangle.html

**Review by chicoxyzzy — APPROVED** [review]

File: src/window/linux.cc [hunk 2]
```diff
@@ -4,6 +4,9 @@ static GtkTargetEntry droppableTypes[] = {
   { (char*) "text/uri-list", 0, 0 }
 };
 
+#define DEFAULT_MONITOR_WIDTH 720
+#define DEFAULT_MONITOR_HEIGHT 364
```
Comment (chicoxyzzy) [inline src/window/linux.cc:8]: actually we should take these values from `windowManager->options.default{Width|height}`

**Comment by lamiazar** [comment]
Screenshot from 2023-04-28 13-06-40

File: src/window/linux.cc (hunk 2 above)
Comment (jwerle) [reply src/window/linux.cc:8]: That won't work if `[window] height = 50%` is set

**Review by lamiazar — APPROVED** [review]
This is now working-I was running the wrong command from the local checkout

File: src/window/linux.cc (hunk 2 above)
Comment (chicoxyzzy) [reply src/window/linux.cc:8]: What was the issue with percents? I don't remember. all values should be in pixels in the end

**Pull request merged**